docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -690,3 +688,27 @@ spec:
   platforms: Windows
   tags: malware, hunting
   contributors: kswagler-rh
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Firewall enabled (macOS)
+  query: SELECT 1 FROM alf WHERE global_state >= 1;
+  description: "Checks if the firewall is enabled."
+  resolution: "In System Preferences, open Security & Privacy, navigate to the Firewall tab and click Turn On Firewall."
+  platforms: macOS
+  tags: hardening, compliance, built-in
+  platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Screen lock enabled via MDM profile (macOS)
+  query: SELECT 1 FROM managed_policies WHERE name='askForPassword' AND value='1';
+  description: "Checks that a MDM profile configures the screen lock."
+  resolution: "Ensure your computer is enrolled to MDM. If it is, contact your administrator and ensure a profile is deployed that will enable the screen lock feature on your Mac."
```

Comment:
I suggest:

> Contact your IT administrator to help you enroll your computer in your organization's MDM. If already enrolled, ask your IT administrator to enable the screen lock feature in the profile configuration.

because a user could be not-enrolled in MDM and/or the IT admin didn't configure it correctly.